documentation/physical_web_service_support.md

```diff
@@ -0,0 +1,5 @@
+## Physical Web Service Support
+
+The Physical Web app comes configured to use Google's internal Physical Web Service (PWS), but you can choose to other services as well. The has built in support for 3 different services; Google's internal PWS,  the production open source PWS, and the development open source PWS. In most cases users will want to select from the Google PWS or the Production PWS.
```

Comment:
"you can choose to other services" (choose other services? choose to use other services?)
"The has built in support" (The app has built in support?)
"Google's internal PWS,  the" (please remove extra space)